static/js/InstanceOverview.tsx

```diff
@@ -28,6 +34,47 @@ const InstanceOverview: FC<Props> = ({ instanceName }) => {
     queryFn: () => fetchInstance(instanceName),
   });
 
+  useEffect(() => {
+    if (instance) {
+      setControls(
+        <div style={{ display: "inline", marginRight: "1rem" }}>
```

Comment:
We should avoid adding CSS in JS. Better fix it with a vanilla class.

I think the problem is the tooltip in the delete button. So you could give the tooltip the right utility class from vanilla, to get correct spacing.

Reply:
PS: Not sure vanilla has utility classes for horizontal spacing. Maybe ask the vanilla pros for advice.

Reply:
No, this is generally handled by the grid.